src/components/ShareExperience/utils.js

```diff
@@ -10,6 +10,26 @@ export const handleBlocks = R.compose(
   R.toPairs,
 );
 
+const convertSections = R.compose(
+  sortById,
+  R.map(ele => ele[1]),
+  R.toPairs,
+  R.map(ele => ({
+    subtitle: ele.subtitle,
+    content: ele.content,
+  })),
+);
+
+const convertInterviewQas = R.compose(
+  sortById,
+  R.map(ele => ele[1]),
+  R.toPairs,
+  R.map(ele => ({
+    question: ele.question,
+    answer: ele.answer,
+  })),
+);
+
```

Comment:
The methods can be named after the return type, e.g. `toSectionArray`, `toInterviewQaArray`